apple/testing/default_runner/ios_xctestrun_runner.template.xctestrun

```diff
@@ -50,5 +48,35 @@ BAZEL_COMMAND_LINE_ARGS_SECTION
 BAZEL_SKIP_TEST_SECTION
 BAZEL_ONLY_TEST_SECTION
   </dict>
+  <key>__xctestrun_metadata__</key>
+  <dict>
```

Comment:
Do you know which versions of Xcode this is supported? Xcode 14 & 15 at least?

Reply:
At least 14 and 15. I haven't tried with sub 14.